### ISSUE TITLE
Item Migration (Iris): effective locations

### ISSUE THREAD

**Issue by wwelling** [TAMULib/mod-data-migration#247]
There are 70 item records according to my cross-check that have a different location in voyager than they do in folio.

```
stk       1d1ca55a-86a1-489b-a645-2d52742c196a  vger    2180238  folio   2180228
blcc,res  6daff586-2421-47c0-9026-e0511347a752  vger    845      folio   866
blcc,stk  752c2901-142b-453f-a353-ab5bc6de7020  vger    49691    folio   49690
media,res 96445779-5afc-4bd6-b3f0-d46b37b7f58f  vger    1962     folio   1940
rs,hdr    9e7086ca-99e3-4928-830f-78a8b88687fc  vger    119579   folio   119576
psel,circ ab68df7a-21b7-45d1-88be-9f53081f534e  vger    141      folio   142
psel,res  e8a5154e-71a3-4daa-a2d6-f4626c99ffe6  vger    81       folio   93
```

I drilled down on the psel,res as an example. Here is some data for the 12 that are in folio but not voyager.

```
select i.item_id,perm_location, temp_location, 
modify_date, item_barcode
from amdb.item i,
amdb.item_barcode ib
where ib.item_id = i.item_id
and i.item_id in (
2325669,
2753095,
2814299,
3355905,
3914721,
4370189,
4370192,
4419617,
4533577,
4550839,
4608456,
4630390);
should be in stk or edms,res  are all in psel,res
   ITEM_ID PERM_LOCATION TEMP_LOCATION MODIFY_DA ITEM_BARCODE
---------- ------------- ------------- --------- ------------------------------
   2325669            32            37 08-DEC-99 A14821047585
   2753095            32            37 17-JAN-08 A14826604413
   2814299            32             0 17-OCT-17 A14826952395
   3355905            32             0           A14832640338
   3914721            32            37 17-OCT-17 A14840147491
   4370189            32            37 21-SEP-18 A14842747744
   4370192            32            37 21-SEP-18 A14842747736
   4419617            32            37 17-OCT-17 A14843208381
   4533577            32            37           A14850201659
   4550839            32             0 28-MAR-19 A14850459901
   4608456            32            37 09-JAN-20 A14850204265
   4630390            32            37           A14851523465
```

**Comment by wwelling** [comment]
@hismith here is what is being inserted into the database for the above items.

```
0cf5d306-7bdb-3592-a55d-9c6f328f788d    {\"id\":\"0cf5d306-7bdb-3592-a55d-9c6f328f788d\",\"hrid\":\"it00000000001\",\"holdingsRecordId\":\"80583476-65d2-30b4-aa6a-6bf71b4e46b5\",\"formerIds\":[\"2325669\"],\"discoverySuppress\":false,\"barcode\":\"A14821047585\",\"effectiveCallNumberComponents\":{\"callNumber\":\"D843 .G23 1997\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[{\"itemNoteTypeId\":\"8d0a5eca-25de-4391-81a9-236eeefdd20b\",\"note\":\"2020:Spring:HIST613600 Cobbs\",\"staffOnly\":true}],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2020-02-07T10:30:17.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.417+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.417+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}  2021-05-13 13:32:27.417Z        95af6e3b-321b-5267-94d5-29a46dc46fd0        80583476-65d2-30b4-aa6a-6bf71b4e46b5    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc    fe050219-8aa1-460d-b40e-930271137175        null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
30a8fcb0-00be-36bf-9d68-df726a9d0d30    {\"id\":\"30a8fcb0-00be-36bf-9d68-df726a9d0d30\",\"hrid\":\"it00000000002\",\"holdingsRecordId\":\"7c3841dc-a92d-3d87-a29b-adb37f4a4fe2\",\"formerIds\":[\"2753095\"],\"discoverySuppress\":false,\"barcode\":\"A14826604413\",\"effectiveCallNumberComponents\":{\"callNumber\":\"HD30.2 .S7835 2000\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2020-01-03T10:30:18.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.482+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.482+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}    2021-05-13 13:32:27.482Z    95af6e3b-321b-5267-94d5-29a46dc46fd0    7c3841dc-a92d-3d87-a29b-adb37f4a4fe2    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc        fe050219-8aa1-460d-b40e-930271137175    null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
5f4d1cc7-e6ed-34c3-95bc-0884d31c6460    {\"id\":\"5f4d1cc7-e6ed-34c3-95bc-0884d31c6460\",\"hrid\":\"it00000000003\",\"holdingsRecordId\":\"706ac9f7-2b65-3394-b040-7391371fb3d4\",\"formerIds\":[\"2814299\"],\"discoverySuppress\":false,\"barcode\":\"A14826952395\",\"effectiveCallNumberComponents\":{\"callNumber\":\"GR105 .B7 1998\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2018-11-14T10:30:21.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"effectiveLocationId\":\"1d1ca55a-86a1-489b-a645-2d52742c196a\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.521+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.521+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}  2021-05-13 13:32:27.521Z        95af6e3b-321b-5267-94d5-29a46dc46fd0    706ac9f7-2b65-3394-b040-7391371fb3d4    5d907602-badb-403f-86fa-35af4c86dc30        null    fe050219-8aa1-460d-b40e-930271137175    null    null    1d1ca55a-86a1-489b-a645-2d52742c196a
1fe1f9d1-fba1-31ef-ab02-200a0c592ae6    {\"id\":\"1fe1f9d1-fba1-31ef-ab02-200a0c592ae6\",\"hrid\":\"it00000000004\",\"holdingsRecordId\":\"2f7a2071-dd1b-3fee-9a7d-26b71ddc0c44\",\"formerIds\":[\"3355905\"],\"discoverySuppress\":false,\"barcode\":\"A14832640338\",\"effectiveCallNumberComponents\":{\"callNumber\":\"BV3785.P74 S46 2005\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2019-06-01T09:30:35.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"effectiveLocationId\":\"1d1ca55a-86a1-489b-a645-2d52742c196a\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.563+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.563+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}     2021-05-13 13:32:27.563Z        95af6e3b-321b-5267-94d5-29a46dc46fd0    2f7a2071-dd1b-3fee-9a7d-26b71ddc0c44    5d907602-badb-403f-86fa-35af4c86dc30        null    fe050219-8aa1-460d-b40e-930271137175    null    null    1d1ca55a-86a1-489b-a645-2d52742c196a
11af77e8-83f9-3b97-824f-c0f28afe1266    {\"id\":\"11af77e8-83f9-3b97-824f-c0f28afe1266\",\"hrid\":\"it00000000005\",\"holdingsRecordId\":\"176c5e53-9d7c-328f-97bd-6c5cde20c6b7\",\"formerIds\":[\"3914721\"],\"discoverySuppress\":false,\"barcode\":\"A14840147491\",\"effectiveCallNumberComponents\":{\"callNumber\":\"GR550 .F284 2009\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2019-03-19T09:30:24.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.591+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.591+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}      2021-05-13 13:32:27.591Z    95af6e3b-321b-5267-94d5-29a46dc46fd0    176c5e53-9d7c-328f-97bd-6c5cde20c6b7    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc        fe050219-8aa1-460d-b40e-930271137175    null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
d9f7119d-64f6-3306-9e8f-0ad9a67808b5    {\"id\":\"d9f7119d-64f6-3306-9e8f-0ad9a67808b5\",\"hrid\":\"it00000000006\",\"holdingsRecordId\":\"98d8af96-d570-3390-871c-6ab44cfd595e\",\"formerIds\":[\"4370189\"],\"discoverySuppress\":false,\"barcode\":\"A14842747744\",\"effectiveCallNumberComponents\":{\"callNumber\":\"GC11.2 .T49 2014\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2020-01-29T10:31:04.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.622+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.622+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}      2021-05-13 13:32:27.622Z    95af6e3b-321b-5267-94d5-29a46dc46fd0    98d8af96-d570-3390-871c-6ab44cfd595e    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc        fe050219-8aa1-460d-b40e-930271137175    null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
3149c660-c834-3414-890f-73cc59258c8a    {\"id\":\"3149c660-c834-3414-890f-73cc59258c8a\",\"hrid\":\"it00000000007\",\"holdingsRecordId\":\"cdfbd2b4-a84d-3573-9f25-a4baccce21a8\",\"formerIds\":[\"4370192\"],\"discoverySuppress\":false,\"barcode\":\"A14842747736\",\"effectiveCallNumberComponents\":{\"callNumber\":\"GC11.2 .T49 2014\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2019-12-11T10:31:07.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.651+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.651+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}      2021-05-13 13:32:27.651Z    95af6e3b-321b-5267-94d5-29a46dc46fd0    cdfbd2b4-a84d-3573-9f25-a4baccce21a8    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc        fe050219-8aa1-460d-b40e-930271137175    null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
cb08ecdb-537d-3584-b397-bcba5cc440e5    {\"id\":\"cb08ecdb-537d-3584-b397-bcba5cc440e5\",\"hrid\":\"it00000000008\",\"holdingsRecordId\":\"b556d1bf-a436-3581-b147-209ac1ef4494\",\"formerIds\":[\"4419617\"],\"discoverySuppress\":false,\"barcode\":\"A14843208381\",\"effectiveCallNumberComponents\":{\"callNumber\":\"B132.Y6 Y53 2007\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"2\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2019-11-20T10:31:03.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.677+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.677+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}      2021-05-13 13:32:27.677Z    95af6e3b-321b-5267-94d5-29a46dc46fd0    b556d1bf-a436-3581-b147-209ac1ef4494    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc        fe050219-8aa1-460d-b40e-930271137175    null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
0601bc58-edba-3f95-ab25-a5b2c3523a7f    {\"id\":\"0601bc58-edba-3f95-ab25-a5b2c3523a7f\",\"hrid\":\"it00000000009\",\"holdingsRecordId\":\"cc6fd18f-58e9-35ed-b254-5019b63c3a86\",\"formerIds\":[\"4533577\"],\"discoverySuppress\":false,\"barcode\":\"A14850201659\",\"effectiveCallNumberComponents\":{\"callNumber\":\"E185.61 .D85 2011\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[{\"itemNoteTypeId\":\"8d0a5eca-25de-4391-81a9-236eeefdd20b\",\"note\":\"history 613 Cobbs, Spring 2020\",\"staffOnly\":true}],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2020-01-24T10:30:56.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.706+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.706+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}     2021-05-13 13:32:27.706Z        95af6e3b-321b-5267-94d5-29a46dc46fd0        cc6fd18f-58e9-35ed-b254-5019b63c3a86    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc    fe050219-8aa1-460d-b40e-930271137175        null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
f80c3817-3099-3ec8-b117-972c424ba493    {\"id\":\"f80c3817-3099-3ec8-b117-972c424ba493\",\"hrid\":\"it00000000010\",\"holdingsRecordId\":\"9eaf8bbe-5cf4-3d7e-a738-f8c928ceedc9\",\"formerIds\":[\"4550839\"],\"discoverySuppress\":false,\"barcode\":\"A14850459901\",\"effectiveCallNumberComponents\":{\"callNumber\":\"E184.S75 C467 1997\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2021-04-25T20:18:15.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"effectiveLocationId\":\"1d1ca55a-86a1-489b-a645-2d52742c196a\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.734+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.734+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}      2021-05-13 13:32:27.734Z        95af6e3b-321b-5267-94d5-29a46dc46fd0    9eaf8bbe-5cf4-3d7e-a738-f8c928ceedc9    5d907602-badb-403f-86fa-35af4c86dc30        null    fe050219-8aa1-460d-b40e-930271137175    null    null    1d1ca55a-86a1-489b-a645-2d52742c196a
67e54a29-8ea0-36ca-bd75-9cfca779f159    {\"id\":\"67e54a29-8ea0-36ca-bd75-9cfca779f159\",\"hrid\":\"it00000000011\",\"holdingsRecordId\":\"77d2bfa9-bddc-30c8-b639-881002196625\",\"formerIds\":[\"4608456\"],\"discoverySuppress\":false,\"barcode\":\"A14850204265\",\"effectiveCallNumberComponents\":{\"callNumber\":\"E184.S75 G655 2011\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2020-01-09T18:02:33.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.772+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.772+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}    2021-05-13 13:32:27.772Z    95af6e3b-321b-5267-94d5-29a46dc46fd0    77d2bfa9-bddc-30c8-b639-881002196625    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc        fe050219-8aa1-460d-b40e-930271137175    null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
2c8feaf9-82ae-3ed5-b985-1ae29cd90794    {\"id\":\"2c8feaf9-82ae-3ed5-b985-1ae29cd90794\",\"hrid\":\"it00000000012\",\"holdingsRecordId\":\"76f51189-8652-3b81-91f0-f541f2baa13e\",\"formerIds\":[\"4630390\"],\"discoverySuppress\":false,\"barcode\":\"A14851523465\",\"effectiveCallNumberComponents\":{\"callNumber\":\"BH39 .A2875 2020\",\"typeId\":\"95467209-6d7b-468b-94df-0f5d7ad2747d\"},\"yearCaption\":[],\"numberOfPieces\":\"1\",\"notes\":[],\"circulationNotes\":[],\"status\":{\"name\":\"Available\",\"date\":\"2020-12-10T10:30:52.000+00:00\"},\"materialTypeId\":\"fe050219-8aa1-460d-b40e-930271137175\",\"permanentLoanTypeId\":\"5d907602-badb-403f-86fa-35af4c86dc30\",\"temporaryLoanTypeId\":\"886d8b42-267d-4308-bab2-3ea56a1df0dc\",\"temporaryLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"effectiveLocationId\":\"96445779-5afc-4bd6-b3f0-d46b37b7f58f\",\"electronicAccess\":[],\"statisticalCodeIds\":[\"38c86b2b-8156-4b7e-943e-460e15ea0dc0\"],\"metadata\":{\"createdDate\":\"2021-05-13T13:32:27.799+00:00\",\"createdByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\",\"updatedDate\":\"2021-05-13T13:32:27.799+00:00\",\"updatedByUserId\":\"95af6e3b-321b-5267-94d5-29a46dc46fd0\"}}      2021-05-13 13:32:27.799Z    95af6e3b-321b-5267-94d5-29a46dc46fd0    76f51189-8652-3b81-91f0-f541f2baa13e    5d907602-badb-403f-86fa-35af4c86dc30    886d8b42-267d-4308-bab2-3ea56a1df0dc        fe050219-8aa1-460d-b40e-930271137175    null    96445779-5afc-4bd6-b3f0-d46b37b7f58f    96445779-5afc-4bd6-b3f0-d46b37b7f58f
```

**Comment by wwelling** [comment]
Unable to reproduce locally from Loan Migration. I suspect some other process. Still investigating.
